src/Logging/messages.lua

```diff
@@ -5,8 +5,10 @@
 ]]
 
 return {
+	attributeNameNil = "Attribute name cannot be nil",
```

Comment:
This error should probably also be checked for `AttributeChange`. Currently, it only applies to `Attribute`.

Reply:
Fixed! I've also moved this one out of the apply function for a clearer traceback.